app/src/main/java/net/bible/android/view/activity/base/DocumentSelectionBase.kt

```diff
@@ -427,11 +427,20 @@ abstract class DocumentSelectionBase(
                         val searchString = "${binding.freeTextSearch.text}*"
                         val osisIds = if(searchString.length < 3) null else dao.search(searchString).toSet()
 
+                        val test = { it: Book ->
+                            // matches from list of osisIds, if any
+                            (osisIds?.contains(it.osisID) ?: (searchString.length < 3)) ||
```

Comment:
instead of osisID should we look for initials?

Reply:
I think that makes more senese. I was just using what was there before.